src/main.rs

```diff
@@ -257,8 +265,13 @@ fn handle_rate_update(
                     new_sell_order,
                     new_buy_order,
                 })) => {
-                    swarm.orderbook.publish(new_sell_order.into());
-                    swarm.orderbook.publish(new_buy_order.into());
+                    if let Err(e) = swarm.make(new_sell_order.into()) {
+                        tracing::error!("Failed to make new order: {}", e)
```

Comment:
I think the terminology is confusing here. We did not fail to make (generate) an order, we failed to publish it.

Reply:
Fair, I'll change that.